README.md

```diff
@@ -11,44 +14,78 @@ Computations and statistics on manifolds with geometric structures.
 - The documentation of ```geomstats``` can be found on the [documentation website](https://geomstats.github.io/).
 - If you find ``geomstats`` useful, please kindly cite our [paper](https://arxiv.org/abs/1805.08308).
 
+## Install geomstats via pip3
 
-## Installation
-
-OS X & Linux:
+From a terminal (OS X & Linux), you can install geomstats and its requirements with ``pip3`` as follows::
 
 ```
 pip3 install -r requirements.txt
 pip3 install geomstats
 ```
 
-Pytorch and tensorflow requirements are optional, as geomstats can be used with numpy only.
+This installs the latest version uploaded on PyPi.
+
+## Install geomstats via Git
+
+From a terminal (OS X & Linux), you can install geomstats and its requirements via Git as follows::
 
-To change backend:
 ```
-export GEOMSTATS_BACKEND=pytorch
+pip3 install -r requirements
+git clone https://github.com/geomstats/geomstats.git
 ```
 
-## Getting started
+This installs the latest GitHub version, useful for developers.
 
-Run example scripts, for example:
+## Choose the backend
+
+You can choose your backend by setting the environment variable ``GEOMSTATS_BACKEND`` to ``numpy``, ``tensorflow`` or ``pytorch``. By default, the numpy backend is used. You should only use the numpy backend for examples with visualizations.
```

Comment:
Not clear (implies that numpy only works for visualization).
--> 'The visualizations are only available with the numpy backend'